.renovaterc.json

```diff
@@ -8,5 +8,17 @@
   "dependencies": {
     "enabled": false
   },
-  "ignorePaths": []
+  "ignorePaths": [],
+  "packageRules": [
+    {
+      "updateTypes": ["patch"],
+      "groupName": "dependencies (patch)",
+      "schedule": "before 8am every weekday"
```

Comment:
Which time zone is this? Maybe make it midnight (PT or ET doesn't matter), so Travis can run when the load is lowest?

Reply:
Done.